gradle.properties

```diff
@@ -15,5 +15,5 @@
 #
 #
 
-micronautCoreVersion=4.8.18
-micronautTestVersion=4.7.0
+micronautCoreVersion=4.4.3
```

Comment:
Regarding this, can you try to disable the transitive deps for micronaut when importing it, e.g.  

```
dependencies {
    implementation('com.example:some-library:1.0') {
        exclude group: 'org.unwanted'
    }
}
```